src/template-tags/tickets.php

```diff
@@ -493,7 +493,7 @@ function tribe_events_has_tickets_on_sale( $event_id ) {
 	 *
 	 * @return string
 	 */
-	function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $ticket ) {
+	function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $ticket, string $sold_label_override = '' ) {
```

Comment:
`@since` should be updated

Reply:
Done.

Reply:
Also created a snapshot test that checks the entire summary on the attendees page.